src/gtfs/read.rs

```diff
@@ -792,31 +791,28 @@ fn make_ntfs_vehicle_journeys(
     // there always is one dataset from config or a default one
     let (_, dataset) = datasets.iter().next().unwrap();
     let mut vehicle_journeys: Vec<objects::VehicleJourney> = vec![];
-    let mut gtfs_tps_ids: HashMap<(u8, u8), u8> = HashMap::new();
-    let mut id_incr: u8 = 0;
+    let mut trip_properties: Vec<objects::TripProperty> = vec![];
+    let mut map_tps_trips: HashMap<(u8, u8), Vec<&Trip>> = HashMap::new();
+    let mut id_incr: u8 = 1;
 
     for t in gtfs_trips {
-        let trip_property_id = match gtfs_tps_ids.entry((t.wheelchair_accessible, t.bikes_allowed))
-        {
-            Vacant(entry) => {
-                id_incr += 1;
-                entry.insert(id_incr)
-            }
-            Occupied(entry) => entry.into_mut(),
-        };
-
-        vehicle_journeys.push(t.to_ntfs_vehicle_journey(
-            routes,
-            dataset,
-            trip_property_id.to_string(),
-        ));
+        map_tps_trips
+            .entry((t.wheelchair_accessible, t.bikes_allowed))
+            .or_insert_with(|| vec![])
+            .push(t);
     }
 
-    let trip_properties: Result<Vec<objects::TripProperty>> = gtfs_tps_ids
-        .iter()
-        .map(|(&(weelchair_id, bike_id), id)| {
-            let trip_property = objects::TripProperty {
-                id: id.to_string(),
+    // If there is no accessibility information in the trips
+    if map_tps_trips.len() == 1 && map_tps_trips.keys().next() == Some(&(0, 0)) {
```

Comment:
Is this really what you want? Maybe you just want to never create the empty property case?